modules/ui/src/styles.scss

```diff
@@ -247,3 +247,21 @@ body:has(app-callout .info):has(app-callout .error)
     opacity: var(--mat-text-button-focus-state-layer-opacity);
   }
 }
+
+.device-item-with-actions[class*='helperbird'],
+.progress-card[class*='helperbird'] {
+  width: max-content !important;
```

Comment:
I am not sure if it a right way to fix the issues with different type of fonts that people can choose and configure. The class `class*='helperbird'` only the class that one tools set to check it.